frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/components/CorrectionsSubmitModal.js

```diff
@@ -20,7 +20,7 @@ function CorrectionsSubmitModal({ t, filingNumber, handleGoToHome }) {
   return (
     <Modal
       actionCancelLabel={t("CS_PRINT_CASE_FILE")}
-      actionCancelOnSubmit={handleGoToHome}
+      actionCancelOnSubmit={() => downloadPdf(tenantId, caseDetails?.additionalDetails?.signedCaseDocument)}
```

Comment:
why download on cancel ?

Reply:
As per the prop requirement for the modal. The actionCancelLabel is the print button so corresponding to it is the on click. @anirudh-0